weasyl/controllers/moderation.py

```diff
@@ -245,3 +245,17 @@ def POST(self):
 
         moderation.edituserconfig(form)
         raise web.seeother("/modcontrol")
+
+
+class modcontrol_auditlog_(controller_base):
+    login_required = True
+    moderator_only = True
+
+    def GET(self):
+        form = web.input(
+            username=None, staff=None, start_date=None, end_date=None)
+
+        return define.webpage(self.user_id, "modcontrol/auditlog.html", [
+            moderation.audit_log(**form),
```

Comment:
Note that `form` includes everything POSTed; the arguments to `web.input` are just defaults. It might be best to pass each argument individually here.